aws-lc-rs/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
 name = "aws-lc-rs"
 authors = ["AWS-LibCrypto"]
-version = "1.6.3"
+version = "1.6.4"
 # this crate re-exports whatever sys crate that was selected
-links = "aws_lc_rs_1_6_3_sys"
+links = "aws_lc_rs_1_6_4_sys"
```

Comment:
sys crate version numbers should be updated too?

Reply:
It should automatically pick up the latest patch version regardless. I'm fine either way.

Reply:
Just looks a little weird to release a version that still refers to the yanked sys crate version. Just removes one less variable to whether the right behavior will occur or not.